synthetics-stats/schema.graphql

```diff
@@ -144,6 +144,7 @@ type PositionFeesInfo @entity(immutable: true) {
   collateralTokenPriceMax: BigInt!
 
   positionFeeAmount: BigInt!
+  liquidationFeeAmount: BigInt!
```

Comment:
liquidationFeeAmount is not available in events prior to newly created ones,
thus exclamation mark is redundant here